src/current/cockroachcloud/use-managed-service-backups.md

```diff
@@ -18,11 +18,19 @@ To access your managed-service backups, select a cluster from the [**Clusters**
 This page describes the **Backup and Restore** page and how to restore your data.
 
 <section class="filter-content" markdown="1" data-scope="serverless">
-Cockroach Labs runs [full cluster backups](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/take-full-and-incremental-backups#full-backups) hourly for every CockroachDB {{ site.data.products.serverless }} cluster. The full backups are retained for 30 days. Once a cluster is deleted, Cockroach Labs retains the full backups for 30 days.
+Cockroach Labs runs [full cluster backups](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/take-full-and-incremental-backups#full-backups) hourly for every CockroachDB {{ site.data.products.serverless }} cluster. Consider the following when using managed-service backups:
+
+- By default, full backups are retained for 30 days. However, if you delete the backup schedule manually you will not be able to restore from these backups.
+- Once a cluster is deleted, Cockroach Labs retains the full backups for 30 days and incremental backups for 7 days.
+
 </section>
 
 <section class="filter-content" markdown="1" data-scope="dedicated">
-Cockroach Labs runs [full cluster backups](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/take-full-and-incremental-backups#full-backups) daily and [incremental cluster backups](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/take-full-and-incremental-backups#incremental-backups) hourly for every CockroachDB {{ site.data.products.dedicated }} cluster. The full backups are retained for 30 days, while incremental backups are retained for 7 days. Once a cluster is deleted, Cockroach Labs retains the full backups for 30 days and incremental backups for 7 days. Backups are stored in the same region that a [single-region cluster]({% link cockroachcloud/plan-your-cluster.md %}#cluster-configuration) is running in or the primary region of a [multi-region cluster](plan-your-cluster.html#multi-region-clusters).
+Cockroach Labs runs [full cluster backups](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/take-full-and-incremental-backups#full-backups) daily and [incremental cluster backups](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/take-full-and-incremental-backups#incremental-backups) hourly for every CockroachDB {{ site.data.products.dedicated }} cluster. Consider the following when using managed-service backups:
+
+- By default, full backups are retained for 30 days, while incremental backups are retained for 7 days. However, if you delete the backup schedule manually or enable [CMEK](https://www.cockroachlabs.com/docs/cockroachcloud/cmek) on the cluster, this will affect the availability of managed backups. Refer to the [CockroachDB Cloud FAQs](https://www.cockroachlabs.com/docs/cockroachcloud/frequently-asked-questions.md#who-is-responsible-for-backups) for more detail.
```

Comment:
bad link

```suggestion
- By default, full backups are retained for 30 days, while incremental backups are retained for 7 days. However, if you delete the backup schedule manually or enable [CMEK](https://www.cockroachlabs.com/docs/cockroachcloud/cmek) on the cluster, this will affect the availability of managed backups. Refer to the [CockroachDB Cloud FAQs](https://www.cockroachlabs.com/docs/cockroachcloud/frequently-asked-questions#who-is-responsible-for-backups) for more detail.
```